cfglib.py

```diff
@@ -305,6 +325,16 @@
                               title='Config service for object storage')
 
 src_objstorage_opts = [
```

Comment:
In commit message you've specified only about identity (keystone) DB config options adding. Remove it from this patch OR add info in the commit message about it please.